design/pv_backup_info.md

```diff
@@ -42,21 +42,23 @@ The 1 version of `VolumeInfo` definition is:
 type VolumeInfoV1 struct {
     PVCName        string    // The PVC's name. The format should be <namespace-name>/<PVC-name>
     PVName         string    // The PV name.
-    BackupMethod   string    // The way the volume data is backed up. The valid value includes `VeleroNativeSnapshot`, `PodVolumeBackup`, `CSISnapshot` and `Skipped`.
+    BackupMethod   string    // The way the volume data is backed up. The valid value includes `VeleroNativeSnapshot`, `PodVolumeBackup` and `CSISnapshot`.
     SnapshotDataMovement bool // Whether the volume's snapshot data is moved to specified storage.
 
+    Skipped         boolean   // Whether the Volume is skipped in this backup.
     SkippedReason   string    // The reason for the volume is skipped in the backup.
     StartTimestamp  *metav1.Time // Snapshot starts timestamp.
 
     CSISnapshotInfo          CSISnapshotInfo
     SnapshotDataMoveInfo     SnapshotDataMoveInfo
     NativeSnapshotInfo       VeleroNativeSnapshotInfo
     PVBInfo                  PodVolumeBackupInfo
+    PVInfo                   PVInfo
 }
 
 // CSISnapshotInfo is used for displaying the CSI snapshot status
 type CSISnapshotInfo struct {
-    SnapshotHandle  string       // The actual snapshot ID. It can be the cloud provider's snapshot ID or the file-system uploader's snapshot.
+    SnapshotHandle  string       // The actual snapshot ID. It's the storage provider's snapshot ID for CSI.
```

Comment:
It is a minor detail but the comment can simply be "Storage provider's snapshot ID for CSI" or similar. No need to say "actual snapshot ID". Same applies to other places as well.

Reply:
Thanks. Modified.